clients/filesystem-fuse/Cargo.toml

```diff
@@ -29,8 +29,13 @@ repository = "https://github.com/apache/gravitino"
 name = "gvfs-fuse"
 path = "src/main.rs"
 
+[lib]
+name="gvfs_fuse"
+
 [dependencies]
+dashmap = "5.5.3"
 futures-util = "0.3.30"
+fuse3 = { version = "0.8.1", "features" = ["tokio-runtime", "unprivileged"] }
```

Comment:
do you plan to use `fuse3`?

Reply:
Yes, the thread mode of `fuse3` is better than `fuser`. It uses multiple threads with asynchronous I/O.

Reply:
does `fuse3` have some disadvantages?

Reply:
`FUSE3` is more modern, with fewer legacy burdens, but it has a smaller user base. On the other hand, `fuser` has been around for a long time, its implementation and technologies are older, but it has a larger user base, making it more stable in practice.

I'm not sure which one would be better to use right now. What do you think?

Reply:
My concern about `FUSE3` is the stability and smaller user community.  It may hard to resolve if encountering some underlying problems.

Reply:
Do you think we should use fuser?